doc/langref.html.in

```diff
@@ -4740,6 +4740,43 @@ fn cmpxchgWeakButNotAtomic(comptime T: type, ptr: *T, expected_value: T, new_val
       </p>
       {#see_also|@cVaArg|@cVaCopy|@cVaEnd#}
       {#header_close#}
+	
+      {#header_open|@deprecated#}
+      <pre>{#syntax#}@deprecated(value: anytype) @TypeOf(value){#endsyntax#}</pre>
+      <pre>{#syntax#}@deprecated() void{#endsyntax#}</pre>
+      <p>
+	  Used to mark a given code path as deprecated. It evaluates to the same value
+	  passed in as argument, or the {#syntax#}void{#endsyntax#} value when given none.
```

Comment:
```suggestion
	  passed in as argument, or to {#syntax#}{}{#endsyntax#} value when no argument is given.
```

Mainly the first bit; "the void value" reads like it might literally mean `void`

Reply:
I've removed the syntax block around `void` but kept the "void value" expression because it's also used in other places in the langref.